### ISSUE TITLE
Train and evaluate Gradient Boosting Classifier using LightGBM

### ISSUE THREAD

**Issue by kgomotsosachane** [The-Elite-Loan-Predictor/loan-predictor#39]
## Objective
Train and evaluate Gradient Boosting Classifier (LightGBM) on cleaned and preprocessed dataset

## Tasks
- [ ] Instantiate and train a gradient boosting model using the training data.
- [ ] Evaluation & Performance Comparison : Extract metrics for both Training and Testing sets
   Metrics: Accuracy, Precision, Recall, F1-Score, and Confusion Matrix.
